BepInEx/Contract/PluginInfo.cs

```diff
@@ -3,7 +3,7 @@
 using System.Linq;
 using BepInEx.Bootstrap;
 
-namespace BepInEx.Contract
+namespace BepInEx
```

Comment:
Any reason for swapping the namespace?

Reply:
Just for consistency, everything else in the folder is in BepInEx namespace. It shouldn't break anything since the class instances were not exposed to plugins before.